src/test/java/uk/ac/ebi/atlas/solr/analytics/AnalyticsSearchServiceIT.java

```diff
@@ -0,0 +1,116 @@
+package uk.ac.ebi.atlas.solr.analytics;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.ac.ebi.atlas.configuration.TestConfig;
+import uk.ac.ebi.atlas.search.SemanticQuery;
+import uk.ac.ebi.atlas.search.SemanticQueryTerm;
+import uk.ac.ebi.atlas.species.Species;
+import uk.ac.ebi.atlas.species.SpeciesFactory;
+
+import javax.inject.Inject;
+
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = TestConfig.class)
+public class AnalyticsSearchServiceIT {
+
+    @Inject
+    private SpeciesFactory speciesFactory;
+
+    @Inject
+    private AnalyticsSearchService subject;
+
+    private SemanticQuery query = SemanticQuery.create("zinc finger");
+    private SemanticQuery condition = SemanticQuery.create("watering");
+    private Species species;
+
+    @Before
+    public void setUp() {
+        species = speciesFactory.create("oryza sativa");
+    }
+
+    @Test
+    public void fetchExperimentTypes1() {
```

Comment:
`fetchExperimentTypes1` to `fetchExperimentTypesForGeneQuery`  code readability?

Reply:
Please change other methods if you think it improves code readability?

Reply:
I renamed this one and the other two `fetchExperimentTypes2` and `fetchExperimentTypes3`.